ansible/roles/system_config/tasks/create_psql_user.yml

```diff
@@ -0,0 +1,3 @@
+- name: Decidim | Create PSQL user
+  shell:
+    cmd: sudo -u postgres psql -c "CREATE USER decidim WITH SUPERUSER CREATEDB CREATEROLE"
```

Comment:
Your pg user should not be exposed outside. Please use a secret one

Reply:
What do you mean by secret one?

Reply:
I meant we should not use a predictible user and expose it on github.

Reply:
Thanks @armandfardeau! I changed the config by replacing with an environment variable to configure on the server